.changesets/fix_fix_content_type_error.md

```diff
@@ -0,0 +1,5 @@
+### Improve error message for invalid `content-type` in subgraph response ([Issue #5359](https://github.com/apollographql/router/issues/5359))
+
+To improve their understandability and debuggability the router now includes more details in error messages caused by invalid `content-type` in subgraph response.
+
```

Comment:
````suggestion
### Improve error message produced when subgraphs responses don't include an expected `content-type` header value ([Issue #5359](https://github.com/apollographql/router/issues/5359))

To improve a common debuggability challenge when a subgraph response doesn't contain an expected `content-type` header value, the error message produced will include additional details about the error.

Examples:

   * ```
     HTTP fetch failed from 'test': subgraph response contains invalid 'content-type' header value \"application/json,application/json\"; expected content-type: application/json or content-type: application/graphql-response+json
      ```
   * ```
     HTTP fetch failed from 'test': subgraph response does not contain 'content-type' header; expected content-type: application/json or content-type: application/graphql-response+json
      ```
````